services/online_exam_service/.env

```
@@ -1,18 +1,18 @@
 # SERVER CONFIG
-PORT=8083
+PORT=3000
 
-# DATABASE CONFIG (kết nối đến course_db)
+# DATABASE CONFIG (k?t n?i d?n course_db)
 DB_HOST=localhost
 DB_PORT=5433
 DB_USER=postgres
 DB_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_PORT key

(UnorderedKey)